### ISSUE TITLE
Should not copy things into the enhanced graph without asking

### ISSUE THREAD

**Issue by ftyers** [keggsmurph21/notatrix#16]
Not sure if this is the right place or if Annotatrix is.

Relations from the basic graph should not be copied without the user explicitly asking. If the enhanced graph only has one relation when the user chooses enhanced layout then that is the only relation that should appear.

**Comment by keggsmurph21** [comment]
Thanks @ftyers, this is the right place for that sort of issue.  Unfortunately, this fix would require a bit of restructuring of internal stuff, so I won't be able to get to this for a while.

Basically the problem is that `notatrix` right now is always keeping track of enhanced dependencies, but it only shows them if the user has "turned on" advanced dependencies.  That means that if enhanced dependencies are "on", there's no way to limit _which_ dependencies should up in the `DEPS` column.

I'm thinking of slightly restructuring these sorts of things in the near future, so I'll keep that in mind!

**Comment by ftyers** [comment]
I refer to the [docs](https://universaldependencies.org/u/overview/enhanced-syntax.html):
> Note that the enhanced graph is not necessarily a supergraph of the basic tree, i.e., the graph is not required to contain all the basic dependency relations. For this reason, all relations of the enhanced graph (also the ones that are present in the basic UD tree) have to be included in the DEPS column of a CoNLL-U file.
This means that the two graphs are separate, there should be perhaps helper functions to help to build the enhanced graph from the basic graph (e.g. "copy all non-conflicting dependencies"), but it should not be automatic, and should not overwrite the existing values in the `DEPS` column.

**Comment by kmurphy4** [comment]
Yeah the current implementation is definitely misbehaving.  Sorry :/
